test/functional/tests/io_class/test_io_class_occupancy_remap.py

```diff
@@ -0,0 +1,157 @@
+
+    with TestRun.step("Add default ioclasses"):
+        ioclass_config.add_ioclass(*str(IoClass.default(allocation="0.00")).split(","))
+
+    with TestRun.step("Add ioclasses for all dirs"):
+        for io_class in io_classes:
+            ioclass_config.add_ioclass(
+                io_class.id,
+                f"directory:{io_class.dir_path}&done",
+                io_class.eviction_prio,
+                f"{io_class.max_occupancy:0.2f}",
+            )
+
+        casadm.load_io_classes(cache_id=cache.cache_id, file=ioclass_config_path)
+
+    with TestRun.step("Reset cache stats"):
+        cache.purge_cache()
+        cache.reset_counters()
+
+    with TestRun.step(f"Create 3 files classified in default ioclass"):
+        for i, io_class in enumerate(io_classes[0:3]):
+            run_io_dir(
+                f"{mountpoint}/{i}", int((io_class.max_occupancy * cache_size) / Unit.Blocks4096)
+            )
+
+    with TestRun.step("Check initial occupancy"):
+        for io_class in io_classes:
+            occupancy = get_io_class_occupancy(cache, io_class.id)
+            if occupancy.get_value() != 0:
+                TestRun.LOGGER.error(
+                    f"Incorrect inital occupancy for ioclass id: {io_class.id}."
+                    f" Expected 0, got: {occupancy}"
+                )
+
+    with TestRun.step("Force remap - move files to created directories and read theirs contents"):
+        for i, io_class in enumerate(io_classes):
+            fs_utils.move(source=f"{mountpoint}/{i}", destination=io_class.dir_path)
+            run_io_dir_read(f"{io_class.dir_path}/{i}")
+
+    with TestRun.step("Check if each ioclass reached it's occupancy limit"):
+        for io_class in io_classes[0:3]:
+            actuall_occupancy = get_io_class_occupancy(cache, io_class.id)
+
+            occupancy_limit = (
+                (io_class.max_occupancy * cache_size)
+                .align_down(Unit.Blocks4096.get_value())
+                .set_unit(Unit.Blocks4096)
+            )
+
+            if not isclose(actuall_occupancy.value, occupancy_limit.value, rel_tol=0.1):
+                TestRun.LOGGER.error(
+                    f"Occupancy for ioclass {io_class.id} does not match. "
+                    f"Limit: {occupancy_limit}, actuall: {actuall_occupancy}"
+                )
+
+
+def get_io_class_occupancy(cache, io_class_id, percent=False):
+    return get_io_class_usage(cache, io_class_id, percent=percent).occupancy
+
+
+def get_io_class_usage(cache, io_class_id, percent=False):
+    return cache.get_io_class_statistics(
+        io_class_id=io_class_id, percentage_val=percent
+    ).usage_stats
+
+
+def run_io_dir(path, num_ios):
+    dd = (
+        Dd()
+        .input("/dev/zero")
+        .output(f"{path}")
+        .count(num_ios)
+        .block_size(Size(1, Unit.Blocks4096))
+    )
+    TestRun.LOGGER.info(f"{dd}")
+    dd.run()
+    sync()
+    drop_caches(DropCachesMode.ALL)
```

Comment:
those methods are the same for each test file. Consider adding some file with common methods used by those tests